app/helpers/projects_helper.rb

```diff
@@ -145,6 +146,13 @@ def project_options_for_status(project)
     end
   end
 
+  def project_options_for_templated
```

Comment:
This belongs into the contract. The contract should then double check whether the provided template_id is included in this set. This might not be strictly necessary for now but it is the general pattern we should follow (available options in the contract -> contract checks whether current value is included in the available options) and will allow us to easily and faultlessly adopt new features, e.g. specific permission to instantiate a template.